tests/js/lucene-tests.js

```diff
@@ -0,0 +1,138 @@
+// Sanity checks for search integration.  As these are read only, we use a single instance for all tests.
+"use strict";
+var fluid = fluid || require("infusion");
+fluid.setLogging(true);
+
+var gpii  = fluid.registerNamespace("gpii");
+
+// We use just the request-handling bits of the kettle stack in our tests, but we include the whole thing to pick up the base grades
+require("../../node_modules/kettle");
+require("../../node_modules/kettle/lib/test/KettleTestUtils");
+
+require("./test-harness");
+require("../lib/sequence");
+require("../lib/saneresponse");
+
+fluid.registerNamespace("gpii.pouch.lucene.tests");
+
+fluid.defaults("gpii.pouch.lucene.tests.caseHolder", {
+    gradeNames: ["fluid.test.testCaseHolder", "autoInit"],
+    // TODO: Review with Antranig.  We are use `assertLeftHand` for these, but the left-handedness is too shallow to inspect actual records.
```

Comment:
Fixed this, and the comment about cleaning up after pouch, which has been fixed after the latest update to `gpii-pouch`.